src/context.cpp

```diff
@@ -87,8 +87,9 @@ namespace Sass {
 
   {
 
-    // add cwd to include paths
-    include_paths.push_back(CWD);
+    // Sass 3.4: The current working directory will no longer be placed onto the Sass load path by default.
```

Comment:
Just realized that this change was done in ruby sass 3.4, not 3.5, therefore I think we should already have that removed. So to speak this is a bug in libsass 3.4. Maybe Or I did not get your comment?